sdk/monitor/azure-monitor-ingestion/samples/sample_send_small_logs.py

```diff
@@ -25,9 +25,4 @@
       }
     ]
 
-response = client.upload(rule_id=rule_id, stream_name=os.environ['LOGS_DCR_STREAM_NAME'], logs=body)
-
-## iterates directly over Tuple[HttpResponseError, JSON]:
-for error, failed_logs in response:
-    # prints nothing if there are no failed_logs (i/e the status is success)
-    print(failed_logs, error)
+client.upload(rule_id=rule_id, stream_name=os.environ['LOGS_DCR_STREAM_NAME'], logs=body)
```

Comment:
If this is a sample - we should demonstrate correct error handling. Now that errors are no longer handled by iterating the result, and no callback is passed in, we should use something like:
```python
try:
    client.upload(...)
except HttpResponseError as e:
    print(f"Upload failed: {e})
```

Reply:
Good call. Updated all examples where an error could be raised.